Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -29,6 +29,16 @@
 #define EOS_CAN_SHUTDOWN
 #endif
 
+//#define EOS_DO_NOT_UNLOAD_SDK_ON_SHUTDOWN
```

Comment:
Would probably be good to remove this. I assume it was here during development for testing purposes.

Reply:
I actually leave these in so that it's easier for plugin developers to turn it on and off quickly without launching the editor.